webhook_server/libs/graphql/unified_api.py

```
@@ -0,0 +1,1253 @@
+        """
+        if not self.graphql_client:
+            await self.initialize()
+
+        expression = f"{ref}:{path}"
+        query, variables = QueryBuilder.get_file_contents(owner, name, expression)
+        result = await self.graphql_client.execute(query, variables)  # type: ignore[union-attr]
+        blob = result["repository"]["object"]
+
+        # Check if file exists
+        if blob is None:
+            raise FileNotFoundError(f"File not found: {path} (ref={ref})")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)